winui/ComboBox/Leading-and-Trailing-View.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Leading and trailing view in WinUI ComboBox control | Syncfusion
+description: Learn how to add leading and trailing view (front and back view) in Syncfusion WinUI ComboBox control and more.
+platform: winui
+control: SfComboBox
+documentation: ug
+---
+
+# Leading and Trailing View in WinUI ComboBox (SfComboBox)
+
+This section explains about the leading and trailing view support available in [ComboBox](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfComboBox.html).The `LeadingView` appears before the `ComboBox` selection area and `TrailingView` appears after the `ComboBox`  selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView`.
```

Comment:
This section explains about the leading and trailing view support available in [ComboBox](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfComboBox.html). The `LeadingView` appears before the `ComboBox` selection area, and `TrailingView` appears after the `ComboBox` selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView.`